private/buf/cmd/buf/internal/internaltesting/internaltesting.go

```diff
@@ -48,8 +48,9 @@ func NewEnvFunc(tb testing.TB) func(string) map[string]string {
 	tempDirPath := tb.TempDir()
 	return func(use string) map[string]string {
 		return map[string]string{
-			useEnvVar(use, "CACHE_DIR"): tempDirPath,
-			"PATH":                      os.Getenv("PATH"),
+			useEnvVar(use, "CACHE_DIR"):  tempDirPath,
+			useEnvVar(use, "CONFIG_DIR"): tempDirPath,
+			"PATH":                       os.Getenv("PATH"),
```

Comment:
Seems reasonable in isolation! Are there constants available for these env var strings?

Reply:
CONFIG_DIR feeds [into the appname interface](https://github.com/bufbuild/buf/blob/main/private/pkg/app/appname/appname.go#L43-L48), as does CACHE_DIR and DATA_DIR. CONFIG_DIR is searched for application configuration, which really means [TLS configuration](https://github.com/bufbuild/buf/blob/main/private/buf/bufapp/bufapp.go#L28-L33).

What other kind of constraints are you thinking of?